package.json

```diff
@@ -14,7 +14,8 @@
     "discord.js": "^11.1.0",
     "dotenv": "^4.0.0",
     "lodash": "^4.17.4",
-    "lowdb": "^0.16.2"
+    "lowdb": "^0.16.2",
+    "nedb": "^1.8.0"

```

Comment:
Should we make these optional dependencies? Then only one could be installed that is being used. I'm not 100% sure on how it works, but can definitely research it a bit.